### PR TITLE
Used numpy 1.13.3 for testing/etc

### PR DESCRIPTION
**Pull request by zduey**
Fixes issue #54

### CODE REVIEW

**Comment by mivade** [comment]
matplotlib is the problem child. But since we aren't using it (yet) with ramutils 2.x, maybe we don't need to depend on it explicitly?

**Comment by mivade** [comment]
At any rate, my plan for HTML-based reporting was to use JS plotting, so we wouldn't need matplotlib at all, in theory. Even if we were using it, mpl2.x has a lot of improvements.

File: test_env.yaml [hunk 1]
```diff
@@ -21,14 +20,14 @@ dependencies:
 - python=2.7.13
 - pywavelets=0.5.2
 - pyzmq=16.0.2
-- scikit-learn=0.18.1
-- scipy=0.18.0
+- scikit-learn=0.19.1
```
Comment (mivade) [inline test_env.yaml]: Is this going to cause any issues re: Ramulator using a different sklearn version?

Comment (zduey) [reply test_env.yaml]: So far as I can tell, no. Ramulator has been able to read the generated config, which I assume entails loading the classifier. Definitely worth some further exploration though.

Comment (mivade) [reply test_env.yaml]: I'd suggest running a session of PS4 and running the posthoc tests to see if results differ.

**Comment by mivade** [comment]
Merging, but we should verify that Ramulator doesn't crap out.

**Pull request merged**